Cargo.toml

```diff
@@ -36,9 +37,7 @@ serde = { version = "1.0.188", default-features = false, features = ["derive"] }
 serde_json = "1.0.105"
 thiserror = { version = "1.0.44", default-features = false }
 tokio = { version = "1.30.0", default-features = false, features = [
-  "fs",
-  "macros",
-  "rt-multi-thread",
+  "full",
```

Comment:
We pull in the same amount of deps whether this is trimmed down to the previous list, or the "full" set.